### ISSUE TITLE
Botania pixie registration cancelling causing crash on being hit with full set of Elementium

### ISSUE THREAD

**Issue by Partonetrain** [Partonetrain/rpgattr#3]
```
[21:42:28] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Ticking entity
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:901) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:824) ~[client-intermediary.jar:?]
	at net.minecraft.class_1132.method_3748(class_1132.java:105) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:671) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:265) ~[client-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "net.minecraft.class_1291.method_42127()" because "$$0" is null
	at net.minecraft.class_1293.<init>(class_1293.java:52) ~[client-intermediary.jar:?]
	at net.minecraft.class_1293.<init>(class_1293.java:48) ~[client-intermediary.jar:?]
	at net.minecraft.class_1293.<init>(class_1293.java:44) ~[client-intermediary.jar:?]
	at vazkii.botania.common.handler.PixieHandler.onDamageTaken(PixieHandler.java:81) ~[Botania-1.20.1-445-FABRIC.jar:?]
	at net.minecraft.class_1657.args$bmf000$botania$onHurt(class_1657.java:11452) ~[client-intermediary.jar:?]
	at net.minecraft.class_1657.method_5643(class_1657.java:928) ~[client-intermediary.jar:?]
	at net.minecraft.class_3222.method_5643(class_3222.java:732) ~[client-intermediary.jar:?]
	at net.minecraft.class_1308.method_6121(class_1308.java:1450) ~[client-intermediary.jar:?]
	at net.minecraft.class_1642.method_6121(class_1642.java:347) ~[client-intermediary.jar:?]
	at net.minecraft.class_1366.method_6288(class_1366.java:152) ~[client-intermediary.jar:?]
	at net.minecraft.class_1366.method_6268(class_1366.java:144) ~[client-intermediary.jar:?]
	at net.minecraft.class_1396.method_6268(class_1396.java:28) ~[client-intermediary.jar:?]
	at net.minecraft.class_4135.method_6268(class_4135.java:65) ~[client-intermediary.jar:?]
	at net.minecraft.class_1355.method_38849(class_1355.java:120) ~[client-intermediary.jar:?]
	at net.minecraft.class_1355.method_6275(class_1355.java:111) ~[client-intermediary.jar:?]
	at net.minecraft.class_1308.method_6023(class_1308.java:771) ~[client-intermediary.jar:?]
	at net.minecraft.class_1309.method_6007(class_1309.java:2673) ~[client-intermediary.jar:?]
	at net.minecraft.class_1308.method_6007(class_1308.java:550) ~[client-intermediary.jar:?]
	at net.minecraft.class_1588.method_6007(class_1588.java:44) ~[client-intermediary.jar:?]
	at net.minecraft.class_1642.method_6007(class_1642.java:262) ~[client-intermediary.jar:?]
	at net.minecraft.class_1309.method_5773(class_1309.java:2419) ~[client-intermediary.jar:?]
	at net.minecraft.class_1308.method_5773(class_1308.java:351) ~[client-intermediary.jar:?]
	at net.minecraft.class_1642.method_5773(class_1642.java:235) ~[client-intermediary.jar:?]
	at net.minecraft.class_3218.method_18762(class_3218.java:739) ~[client-intermediary.jar:?]
	at net.minecraft.class_1937.method_18472(class_1937.java:480) ~[client-intermediary.jar:?]
	at net.minecraft.class_3218.method_31420(class_3218.java:385) ~[client-intermediary.jar:?]
	at net.minecraft.class_5574.method_31791(class_5574.java:54) ~[client-intermediary.jar:?]
	at net.minecraft.class_3218.method_18765(class_3218.java:353) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:897) ~[client-intermediary.jar:?]
	... 5 more
```

**Comment by Partonetrain** [comment]
"Fixed" with 2.1.2. Should no longer crash, but it uses an overwrite, so a better solution is needed

**Comment by Partonetrain** [comment]
Use 2.1.2 if using Botania 445 or earlier
Use 2.1.1 if using BotaniaCombat, Rpgattr, and Botania 445 or earlier (and just avoid wearing a full set of Elementium)
Use 2.2.0 with Botania 446+ (unreleased as of writing)
